store/mockstore/unistore/cophandler/analyze.go

```diff
@@ -350,3 +359,123 @@ func (e *analyzeColumnsExec) NewChunk() *chunk.Chunk {
 func (e *analyzeColumnsExec) Close() error {
 	return nil
 }
+
+func handleAnalyzeMixedReq(dbReader *dbreader.DBReader, rans []kv.KeyRange, analyzeReq *tipb.AnalyzeReq, startTS uint64) (*coprocessor.Response, error) {
+	colExec, builder, _, err := buildBaseAnalyzeColumnsExec(dbReader, rans, analyzeReq, startTS)
+	if err != nil {
+		return nil, err
+	}
+	e := &analyzeMixedExec{
+		analyzeColumnsExec: *colExec,
+		colLen:             int(analyzeReq.IdxReq.NumColumns),
+		statsBuilder:       statistics.NewSortedBuilder(flagsToStatementContext(analyzeReq.Flags), analyzeReq.IdxReq.BucketSize, 0, types.NewFieldType(mysql.TypeBlob)),
+	}
+	if analyzeReq.IdxReq.CmsketchDepth != nil && analyzeReq.IdxReq.CmsketchWidth != nil {
+		e.cms = statistics.NewCMSketch(*analyzeReq.IdxReq.CmsketchDepth, *analyzeReq.IdxReq.CmsketchWidth)
+	}
+	builder.RecordSet = e
+
+	collectors, _, err := builder.CollectColumnStats()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	// columns
+	colResp := &tipb.AnalyzeColumnsResp{}
+	for _, c := range collectors {
+		colResp.Collectors = append(colResp.Collectors, statistics.SampleCollectorToProto(c))
+	}
+	// common handle
+	hg := statistics.HistogramToProto(e.statsBuilder.Hist())
+	var cm *tipb.CMSketch
+	if e.cms != nil {
+		cm = statistics.CMSketchToProto(e.cms)
+	}
+	commonHandleResp := &tipb.AnalyzeIndexResp{Hist: hg, Cms: cm}
+	resp := &tipb.AnalyzeMixedResp{
+		ColumnsResp: colResp,
+		IndexResp:   commonHandleResp,
+	}
+	data, err := proto.Marshal(resp)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	return &coprocessor.Response{Data: data}, nil
+}
+
+type analyzeMixedExec struct {
+	analyzeColumnsExec
+
+	colLen       int
+	statsBuilder *statistics.SortedBuilder
+	cms          *statistics.CMSketch
+	rowBuf       []byte
+}
+
+func (e *analyzeMixedExec) Process(key, value []byte) error {
+	// common handle
+	values, _, err := tablecodec.CutCommonHandle(key, e.colLen)
+	if err != nil {
+		return err
+	}
+	e.rowBuf = e.rowBuf[:0]
+	for _, val := range values {
+		e.rowBuf = append(e.rowBuf, val...)
+		if e.cms != nil {
+			e.cms.InsertBytes(e.rowBuf)
+		}
+	}
+	rowData := safeCopy(e.rowBuf)
+	err = e.statsBuilder.Iterate(types.NewBytesDatum(rowData))
+	if err != nil {
+		return err
+	}
+
+	// columns
+	handle, err := tablecodec.DecodeRowKey(key)
```

Comment:
Can we just call `e.analyzeColumnsExec.Process`?

Reply:
Of course, we can.